test/jit/test_logging.py

```diff
@@ -0,0 +1,113 @@
+import os
+import sys
+
+import torch
+
+# Make the helper files in test/ importable
+pytorch_test_dir = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
```

Comment:
Could you test that this is still runnable in Fbcode ? As far as I recall this doesn't work with buck, and it's a reason why onnx tests are not currently runnable in fbcode

Reply:
it is still runnable from fbcode. It's not hard to set up, just no one did it for onnx